.github/dockerfiles/ov_build/ubuntu_22_04_arm64/Dockerfile

```diff
@@ -18,6 +18,7 @@ RUN apt-get update && \
     add-apt-repository --yes --no-update ppa:deadsnakes/ppa && \
     apt-get update && \
     apt-get install \
+        --no-install-recommends \
```

Comment:
```suggestion
        --no-install-recommends \
```
```suggestion
```

Reply:
this property had to be set globally (line10)

Reply:
Thanks, removed